src/azure-cli/azure/cli/command_modules/hdinsight/commands.py

```diff
@@ -92,3 +98,9 @@ def load_command_table(self, _):
         g.show_command('show', 'get_monitoring_status')
         g.custom_command('enable', 'enable_hdi_monitoring')
         g.command('disable', 'disable_monitoring')
+
+    # VirtualMachine operations
+    with self.command_group('hdinsight host', hdinsight_virtual_machines_sdk,
+                            client_factory=cf_hdinsight_virtual_machines) as g:
+        g.command('list', 'list_hosts')
+        g.command('restart', 'restart_hosts', confirmation=True, supports_no_wait=True)
```

Comment:
If support no wait, could we add wait command to wait till condition matched?

Reply:
> If support no wait, could we add wait command to wait till condition matched?

Hi @Juliehzl  Thanks for reminder. I have added the wait command.

Reply:
> > If support no wait, could we add wait command to wait till condition matched?
> 
> Hi @Juliehzl Thanks for reminder. I have added the wait command.

Hi @Juliehzl  As we talked I removed `support_no_wait`, Thank you very much!